### PR TITLE
toast message, bottom panel, step card updates

### PR DESCRIPTION
**Pull request by Alvareon**
1. Added new toast message:

2. Added icon and text selection on the bottom panel:

3. Updated the step highlighting:

4. Minor fixes of UI.

### CODE REVIEW

**Review by cawke — CHANGES_REQUESTED** [review]
pleace change name of PR. this should show us more detailed information

**Review by cawke — CHANGES_REQUESTED** [review]
image

image

File: Assets/MirageXR/Tests/NewUI/ProfileView.cs [hunk 1]
```diff
@@ -295,4 +295,10 @@ private void UpdateConectedLRS(DBManager.LearningRecordStores publicCurrentLearn
                 break;
         }
     }
+
+    private void CloseAndReturnToHomePage()
+    {
+        Close();
+        RootView_v2.Instance.ReturnToHomePage();
+    }
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/ProfileView.cs]: why do we have to go back to the home screen instead of just closing the profile window?

**Review by cawke — CHANGES_REQUESTED** [review]

File: Assets/MirageXR/Tests/NewUI/ProfileView.cs (hunk 1 above)
Comment (Alvareon) [reply Assets/MirageXR/Tests/NewUI/ProfileView.cs]: to switch the toggle on the bottom panel

Comment (fwild) [reply Assets/MirageXR/Tests/NewUI/ProfileView.cs]: @cawke happy with the answer?

Comment (Alvareon) [reply Assets/MirageXR/Tests/NewUI/ProfileView.cs]: fixed

**Comment by Alvareon** [comment]
> image
> 
> image

fixed

**Review by cawke — CHANGES_REQUESTED** [review]
Delete the old version

image

**Review by cawke — APPROVED** [review]

**Pull request merged**